lib/actions/client/preloader-handler.js

```diff
@@ -30,28 +30,28 @@ class PreloaderHandler
         );
     }
 
-    loadContents(uiScene)
+    loadContents(scenePreloader)
```

Comment:
this scenePreloader should be an instance of our own ScenePreloaderDriver, that way you can avoid calling getSceneDriver(), that method should be removed

Reply:
yeap! That will be done on the next step, when I extract functionallity of the sceneprelaoder into BaseScenePreloader.